#! /bin/sh /usr/share/dpatch/dpatch-run
## 81_clean_up_also_backup_files <otto@seravo.fi>
##

@DPATCH@
=== modified file 'cmake/install_macros.cmake'
--- old/cmake/install_macros.cmake	2014-01-22 23:02:37 +0000
+++ new/cmake/install_macros.cmake	2014-02-10 09:00:11 +0000
@@ -397,6 +397,7 @@
       PATTERN "*.vcxproj.filters" EXCLUDE
       PATTERN "*.vcxproj.user" EXCLUDE
       PATTERN "CTest" EXCLUDE
+      PATTERN "*~" EXCLUDE
     )
   ENDIF()
 ENDFUNCTION()
